### PR TITLE
Karenswry external readmes

### PR DESCRIPTION
**Pull request by karenswry**
Ruby gems and iOS pods-related changes + a touch of T1345 (slight changes in bug report and pull request templates).

## Checklist

- [X] Change is covered by automated tests
- [X] The [coding guidelines] are followed
- [X] Example projects and code samples are updated (in case of API changes)

[coding guidelines]: https://github.com/cossacklabs/themis/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by karenswry** [comment]
Thank you for the review! Hmm, I could be biased, too as I keep seeing the word "wrapper" with the very same meaning that we attach to it, across numerous unrelated projects. 
And yes, introducing minor doc updates without giving everyone this much trouble would be super cool. But we use what we have.

File: src/wrappers/themis/ruby/rbthemis.gemspec [hunk 1]
```diff
@@ -3,7 +3,7 @@ Gem::Specification.new do |s|
   s.version     = '0.12.0'
   s.date        = '2019-09-26'
   s.summary     = 'Data security library for network communication and data storage for Ruby'
-  s.description = 'Themis is a data security library, providing users with high-quality security services for secure messaging of any kinds and flexible data storage. Themis is aimed at modern developers, with high level OOP wrappers for Ruby, Python, PHP, Java / Android and iOS / OSX. It is designed with ease of use in mind, high security and cross-platform availability.'
+  s.description = 'Themis is a convenient cryptographic library for data protection. It provides secure messaging with forward secrecy and secure data storage. Themis is aimed at modern development practices and has a unified API and wrappers across 12 platforms, including Ruby, JavaScript, iOS/macOS, Python, and Java/Android.'
```
Comment (vixentael) [inline src/wrappers/themis/ruby/rbthemis.gemspec]:
```suggestion
  s.description = 'Themis is a convenient cryptographic library for data protection. It provides secure messaging with forward secrecy and secure data storage. Themis is aimed at modern development practices and has a unified API across 12 platforms, including Ruby, JavaScript, iOS/macOS, Python, and Java/Android.'
```

File: themis.podspec [hunk 2]
```diff
@@ -2,7 +2,7 @@ Pod::Spec.new do |s|
     s.name = "themis"
     s.version = "0.12.2"
     s.summary = "Data security library for network communication and data storage for iOS and mac OS"
-    s.description = "Themis is a data security library, providing users with high-quality security services for secure messaging of any kinds and flexible data storage. Themis is aimed at modern development practices, with high level OOP wrappers for iOS / macOS, node,js, Go, Ruby, Python, PHP and Java / Android. It is designed with ease of use in mind, high security and cross-platform availability."
+    s.description = "Themis is a convenient cryptographic library for data protection. It provides secure messaging with forward secrecy and secure data storage. Themis is aimed at modern development practices and has a unified API and wrappers across 12 platforms, including iOS/macOS, Ruby, JavaScript, Python, and Java/Android."
```
Comment (vixentael) [inline themis.podspec]:
```suggestion
    s.description = "Themis is a convenient cryptographic library for data protection. It provides secure messaging with forward secrecy and secure data storage. Themis is aimed at modern development practices and has a unified API across 12 platforms, including iOS/macOS, Ruby, JavaScript, Python, and Java/Android."
```

**Comment by karenswry** [comment]
Done here. Please merge.

**Pull request merged**